resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml

```diff
@@ -63,6 +69,109 @@ Cura.ExpandablePopup
                         anchors.verticalCenter: parent.verticalCenter
                     }
 
+                    MouseArea // Connection status tooltip hover area
+                    {
+                        id: tooltipHoverArea
+                        anchors.fill: parent
+                        hoverEnabled: true //getConnectionStatusMessage() !== ""
```

Comment:
This was some old code? Maybe the commented code should be removed. Or maybe you want to replace it with this:
```suggestion
                        hoverEnabled: tooltip.text !== ""
```
Or this:
```suggestion
                        hoverEnabled: badge.source !== ""
```

After all, you only want the tooltip to appear when there is an icon.

I'd be fine with either of these 3 suggestions.

Reply:
Indeed, this is leftover code from copy/pasting. Thanks for spotting it.

Reply:
Fixed in c805aac